safe_transaction_service/tokens/tests/test_price_service.py

```diff
@@ -105,7 +105,7 @@ def test_get_native_coin_usd_price(
 
         # Aurora
         price_service.ethereum_network = EthereumNetwork.AURORA
-        with mock.patch.object(CoingeckoClient, "get_aoa_usd_price", return_value=1.3):
+        with mock.patch.object(BinanceClient, "get_aurora_usd_price", return_value=1.3):
```

Comment:
Out of curiosity, why do you change coingecko by binance?

Reply:
Because now `Binance` will be the first oracle to be used